code/game/objects/structures/gate.dm

```diff
@@ -123,6 +147,22 @@ GLOBAL_LIST_EMPTY(biggates)
 	isSwitchingStates = FALSE
 	update_icon()
 
+/obj/structure/gate/proc/crush(mob/living/crushed_mob)
+	crushed_mob.gib()
+
+/obj/structure/gate/bars/crush(mob/living/crushed_mob)
+	if(iscarbon(crushed_mob))
+		var/mob/living/carbon/crushed_carbon = crushed_mob
+		for(var/limb_index in bodyparts_to_crush)
+			var/obj/item/bodypart/limb_to_crush = crushed_carbon.get_bodypart(limb_index)
+			if(limb_to_crush)
+				var/random_number = rand(120, 300)
+				crushed_carbon.apply_damage(crush_damage_multiplier * random_number, BRUTE, limb_to_crush, crushed_carbon.run_armor_check(limb_to_crush, "blunt"))
+				limb_to_crush.try_crit(BCLASS_STAB, random_number)
```

Comment:
try to reduce the chance of crit . the damage or something . it basically kills you instantly when crushing even when using armor